_posts/2019-03-15-bridge.md

```diff
@@ -0,0 +1,33 @@
+---
+site: freiburg
+title: "New Paper - Biomolecular Reaction and Interaction Dynamics Global Environment (BRIDGE)"
+tags: [paper]
+doi: "10.1093/bioinformatics/btz107"
+supporters:
+ - denbi
```

Comment:
Wasn't sure what to add here, so put the same as for the most recent paper post.